python/meter-health/meter_health.py

```diff
@@ -86,15 +87,26 @@ def request_organizations(self):
         del self.payload['expand']
         return response.json()
 
-    def retrieve_meter_info_for(self, org_id):
+    def retrieve_meter_info_for(self, org_id, try_number):
         # Requests meters for an organization, if they aren't meters it returns []
-        response = requests.get(self.generate_request_url(self.METERS % org_id), params=self.payload)
-        json_response = response.json()
-        return json_response['embedded']['meters'] if self.retrieve_meters_conditions(json_response) else []
+        if try_number < 20:
```

Comment:
You could create a constant class to handle this value. It can be easily changed anytime.